recipes/rwa-python/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "0.8.5" %}
+
+package:
+  name: rwa-python
+  version: {{ version }}
+
+source:
+  url: https://github.com/DecBayComp/RWA-python/archive/v{{ version }}.tar.gz
+  sha256: 9bcc928ccfb22d1353d59a95c98062ef2b6543c56044ee92cf52ded32cb6d262
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install ."
+
+requirements:
+  host:
+    - h5py
+    - numpy
+    - pandas
+    - pip
+    - python
+    - scipy
+    - six
+  run:
+    - hdf5
+    - h5py
+    - numpy
+    - pandas
+    - python
+    - scipy
+    - six
+
+test:
+  imports:
+    - rwa
+  requires:
+    - pytest
+  commands:
+    - pytest tests/
+  source_files:
+    - tests/test_*.py
+
+about:
+  home: https://github.com/DecBayComp/RWA-python
+  license: Apache-2.0
+  license_family: APACHE
+  license_file: LICENSE
+  summary: "HDF5-based serialization library for Python datatypes"
+  doc_url: https://rwa-python.readthedocs.io/
+
+extra:
+  recipe-maintainers:
+    - DecBayComp
```

Comment:
This should normally contain your username. Adding others as well is fine but we need some kind of confirmation from them (normally a comment or :+1: on the PR)